be/src/exec/aggregator.cpp

```diff
@@ -759,13 +757,15 @@ Status Aggregator::output_chunk_by_streaming_with_selection(Chunk* input_chunk,
 }
 
 void Aggregator::try_convert_to_two_level_map() {
-    if (_mem_tracker->consumption() > two_level_memory_threshold) {
+    auto current_size = _hash_map_variant.reserved_memory_usage(mem_pool());
```

Comment:
why replace _mem_tracker->consumption() with  _hash_map_variant.reserved_memory_usage(mem_pool())?

Reply:
because the _mem_tracker here is unnecessary, it will be set to `_hash_map_variant.reserved_memory_usage(mem_pool())` everytime calling `try_convert_to_two_level_map`, so I just remove it.